### ISSUE TITLE
Delegate vote

### ISSUE THREAD

**Issue by Akay74** [y-yildirim/08-Tokenized-Votes#2]
https://github.com/y-yildirim/08-Tokenized-Votes/blob/b0f9ac4a57f5a8dcf4f9ce1f1b19925c4605b7cf/Project/scripts/vote.ts#L26-L28
https://github.com/y-yildirim/08-Tokenized-Votes/blob/b0f9ac4a57f5a8dcf4f9ce1f1b19925c4605b7cf/Project/scripts/vote.ts#L37-L40

Before casting a vote, aren't we supposed to delegate voting power to the address? Or is this done inside the delegate script already? Because I implemented the delegate function inside the vote script but after looking at the code I figured I was doing something wrong there.

**Comment by y-yildirim** [comment]
Yes, after minting tokens with `mint` script, we need to delegate using `delegate` script to be able to cast vote. https://github.com/y-yildirim/08-Tokenized-Votes/blob/b0f9ac4a57f5a8dcf4f9ce1f1b19925c4605b7cf/Project/scripts/delegate.ts#L22-L26
